MastodonSDK/Sources/MastodonUI/View/Content/StatusAuthorView.swift

```diff
@@ -152,25 +152,30 @@ extension StatusAuthorView {
     }
 
     public func setupAuthorMenu(menuContext: AuthorMenuContext) -> (UIMenu, [UIAccessibilityCustomAction]) {
-        var actions: [MastodonMenu.Action] = []
-
-        actions = [
-            .muteUser(.init(
-                name: menuContext.name,
-                isMuting: menuContext.isMuting
-            )),
-            .blockUser(.init(
-                name: menuContext.name,
-                isBlocking: menuContext.isBlocking
-            )),
-            .reportUser(
-                .init(name: menuContext.name)
-            ),
+        var actions = [MastodonMenu.Action]()
+
+        if !menuContext.isMyself {
+            actions.append(contentsOf: [
```

Comment:
Form a code-style perspective I would prefer, but that can be refactored later depending on what the team's stance on this is.

So far this LGTM!

```
actions += [
                .muteUser(.init(
                    name: menuContext.name,
                    isMuting: menuContext.isMuting
                )),
                .blockUser(.init(
                    name: menuContext.name,
                    isBlocking: menuContext.isBlocking
                )),
                .reportUser(
                    .init(name: menuContext.name)
                )
            ]
 ```